graylog2-web-interface/src/components/configurations/IndexSetsDefaultsConfig.tsx

```diff
@@ -0,0 +1,237 @@
+  const handleSaveConfig = async (configToSave: IndexConfig) => updateConfig(configToSave);
+
+  useEffect(() => {
+    IndicesConfigurationActions.loadRotationStrategies().then((loadedRotationStrategies) => {
+      setRotationStrategies(loadedRotationStrategies);
+    });
+
+    IndicesConfigurationActions.loadRetentionStrategies().then((loadedRetentionStrategies) => {
+      setRetentionStrategies(loadedRetentionStrategies);
+    });
+  }, []);
+
+  const saveConfig = (values, { setSubmitting }) => {
+    handleSaveConfig(values)
+      .then((config) => {
+        setCurrentConfig(config as IndexConfig);
+        setShowModal(false);
+      })
+      .catch(() => {
+        setSubmitting(false);
+      });
+  };
+
+  const resetConfig = () => {
+    setShowModal(false);
+  };
+
+  if (!rotationStrategies || !rotationStrategies || !currentConfig) {
+    return <Spinner />;
+  }
+
+  const rotationConfig = {
+    config: currentConfig.rotation_strategy_config,
+    strategy: currentConfig.rotation_strategy_class,
+  };
+  const retentionConfig = {
+    config: currentConfig.retention_strategy_config,
+    strategy: currentConfig.retention_strategy_class,
+  };
+
+  const getRotationConfigState = (strategy: string, data: string) => {
+    return {
+      rotation_strategy_config: data,
+      rotation_strategy_class: strategy,
+    };
+  };
+
+  const getRetentionConfigState = (strategy: string, data: string) => {
+    return {
+      retention_strategy_class: strategy,
+      retention_strategy_config: data,
+    };
+  };
+
+  return (
+    <div>
+      <h2>Index Defaults</h2>
+      <p>Defaults for newly created index sets.</p>
+      <StyledDefList>
+        <dt>Index analyzer:</dt>
+        <dd>{currentConfig.index_analyzer}</dd>
+        <dt>Shards per Index:</dt>
+        <dd>{currentConfig.shards}</dd>
+        <dt>Replicas per Index:</dt>
+        <dd>{currentConfig.replicas}</dd>
+        <dt>Index optimization disabled:</dt>
+        <dd>{currentConfig.index_optimization_disabled ? 'Yes' : 'No'}</dd>
+        <dt>Max. Number of Segments:</dt>
+        <dd>{currentConfig.index_optimization_max_num_segments}</dd>
+        <dt>Field type refresh interval:</dt>
+        <dd>{currentConfig.field_type_refresh_interval} {lodash.capitalize(currentConfig.field_type_refresh_interval_unit)}</dd>
+        <br />
+        <IndexMaintenanceStrategiesSummary config={rotationConfig}
+                                           pluginExports={PluginStore.exports('indexRotationConfig')} />
+        <IndexMaintenanceStrategiesSummary config={retentionConfig}
+                                           pluginExports={PluginStore.exports('indexRetentionConfig')} />
+      </StyledDefList>
+
+      <p>
+        <IfPermitted permissions="indices:changestate">
+          <Button bsStyle="info"
+                  bsSize="xs"
+                  onClick={() => {
+                    setShowModal(true);
+                  }}>Update
```

Comment:
nit: every other configuration button on the `System > Configurations` page says `Edit Configuration`. To keep it consistent, we may want to reuse that wording here too. 
![image](https://user-images.githubusercontent.com/91903901/204906697-2d9d98a4-8793-44b6-af8d-66b8b3219d07.png)

Reply:
Great catch. Fixed.